schema/gendummydata.py

```diff
@@ -18,6 +18,18 @@
 
 import bcrypt
 
+def getosname():
+    if (os.name == 'posix'):
+        try:
+            with open('/etc/os-release', 'r') as f:
+                for lines in f:
+                    if(lines.find('NAME') > -1):
+                        osname = lines.split('=')[1].strip()
+        except:
```

Comment:
Can you make this catch a specific exception instead of all exceptions? I know this makes it more likely to be fault-tolerant, but if we only catch certain exceptions it'll make it easier to catch errors in the code.